components/clp-py-utils/clp_py_utils/start-spider-worker.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import logging
+import os
+import pathlib
+import socket
+import subprocess
+
+# Setup logging
+# Create logger
+logger = logging.getLogger("start-spider-worker")
+logger.setLevel(logging.INFO)
+# Setup console logging
+logging_console_handler = logging.StreamHandler()
+logging_formatter = logging.Formatter("%(asctime)s [%(levelname)s] %(message)s")
+logging_console_handler.setFormatter(logging_formatter)
+logger.addHandler(logging_console_handler)
+
+
+def parse_args() -> argparse.Namespace:
+    """
+    Parses command line arguments.
+    :return: The parsed arguments.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--concurrency", type=int, default=1, help="Number of concurrent spider workers."
```

Comment:
```suggestion
        "--concurrency", type=int, default=1, help="Number of concurrent Spider workers."
```
Please check all docstring/comments to ensure we use `Spider` not `spider`.